### PR TITLE
Archand sam virtual navigation menu

### PR DESCRIPTION
**Pull request by archiemartini**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,27 +1,12 @@
 {
```
Comment (tomoneill32) [inline .eslintrc.json]: I don't think this should be in the commit, everyone's lint is different, not relevant to your feature. Maybe add to gitignore then a new commit?

File: .gitignore [hunk 2]
```diff
@@ -63,3 +63,5 @@ typings/
 # cypress.io
 cypress/screenshots
 cypress/videos
+
+node_modules
```
Comment (tomoneill32) [inline .gitignore]: already in gitignore on line 36, can remove this line

File: acebook@0.0.0 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (tomoneill32) [inline acebook@0.0.0]: I think this should also be in gitignore and not merged to main

File: views/home/index.hbs [hunk 4]
```diff
@@ -1,2 +1,4 @@
+<center>
```
Comment (gawainhewitt) [inline views/home/index.hbs:1]: the <center> tag is not supported in HTML5 - centering should be managed by css properties instead?

File: cypress/integration/nav_bar_home_button_spec.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+describe("Navbar home button", () => {
+  it("navigates to the home page", () => {
+    cy.visit("/users/new");
+    cy.get("#home").click();
+    cy.url().should("include", "/?");
```
Comment (tomoneill32) [inline cypress/integration/nav_bar_home_button_spec.js]: I think the test should be equal http://localhost:3030/ , simon knows the exact syntax

File: .eslintrc.json (hunk 1 above)
Comment (gawainhewitt) [inline .eslintrc.json]: I think this should be in .gitignore

File: acebook@0.0.0 (hunk 3 above)
Comment (gawainhewitt) [inline acebook@0.0.0]: not clear what this is?

File: cypress/integration/nav_bar_logout_spec.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+     // sign in
+     cy.visit("/sessions/new");
+     cy.get("#email").type("someone@example.com");
+     cy.get("#password").type("password");
+     cy.get("#submit").click();
+     cy.url().should("include", "/posts")
+     // click log-out
+    
+    cy.get("#log-out").click();
+    cy.url().should("include", "/sessions/new");
```
Comment (tomoneill32) [inline cypress/integration/nav_bar_logout_spec.js]: This should be redirecting to home on log out, though it doesn't in your branch. We can probably change the test after merging.

File: eslint [hunk 7]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (tomoneill32) [inline eslint]: Add to gitignore I think

File: views/layout.hbs [hunk 8]
```diff
@@ -3,11 +3,37 @@
...
+          </form>
+         </div>
+        </div>
+        </div>
+      
+      
+    </header>
+
     {{{body}}}
+    <script src="https://kit.fontawesome.com/d88bffd7d7.js" crossorigin="anonymous"></script>
```
Comment (gawainhewitt) [inline views/layout.hbs:30]: cam i know more about this please? Is this to access custom fonts? If so do we have some backups?

**Review by gawainhewitt — CHANGES_REQUESTED** [review]
Great work team

I had a couple of questions which would be good to discuss please - including some clarifications and some change requests

Thanks

**Review by tomoneill32 — COMMENTED** [review]
Lint stuff to go on gitignore, change maybe one test. CSS stuff looks good.

File: views/home/index.hbs (hunk 4 above)
Comment (SamButton12) [reply views/home/index.hbs:1]: help

Comment (Rmorbey) [reply views/home/index.hbs:1]: Can you help us implement that, thanks.

File: views/layout.hbs (hunk 8 above)
Comment (SamButton12) [reply views/layout.hbs:30]: This line is for the home icon in the navbar. It is being used as a placeholder until we have chosen to include/not include a home/logo icon to link to the home page. 

It is rendering the icon from an outside website.

File: views/layout.hbs [hunk 9]
```diff
@@ -3,11 +3,37 @@
...
   </head>
   <body>
-    <form action='/sessions?_method=DELETE' method='POST'>
-      <input id="logout" type="submit" value="Log Out">
-    </form>
+
+    <header>
+      <div class="navbar">
+        <div class="header_buttons">
+          <form action='/' method='GET'>
```
Comment (simon-2357) [inline views/layout.hbs]: `<a href="/"><i class=whatever"> Home</a>`

File: cypress/integration/nav_bar_home_button_spec.js (hunk 5 above)
Comment (simon-2357) [inline cypress/integration/nav_bar_home_button_spec.js]: use eq and whole URL

File: cypress/integration/nav_bar_logout_spec.js (hunk 6 above)
Comment (simon-2357) [inline cypress/integration/nav_bar_logout_spec.js]: same as eq "/"

**Review by simon-2357 — CHANGES_REQUESTED** [review]

**Pull request merged**